chef/cookbooks/updater/recipes/default.rb

```diff
@@ -40,6 +40,31 @@
       zypper_command += " --auto-agree-with-licenses"
     end
 
+    ruby_block "check for HA updates" do
+      block do
+        case node[:updater][:zypper][:method]
+        when "patch"
+          command = "zypper list-patches"
+        when "update"
+          command = "zypper list-updates"
+        when "dist-upgrades"
+          # TODO: check the correct one for here
+          command = "zypper list-updates"
+        end
+        command += '|egrep "corosync|pacemaker" -q'
+        `#{command}`
+        node.run_state["found_ha_packages"] = $?.exitstatus
+      end
+      not_if { search(:node, "roles:pacemaker-cluster-member").empty? }
+    end
+
+    ["corosync", "pacemaker"].each do |s|
```

Comment:
No, I was wrong.  It's enough to stop corosync, [as you already noted](https://trello.com/c/13FOsgtH/13-bug-983617-failed-to-apply-updater-barclamp-when-stopping-pacemaker#comment-578f9d8897f261607988498c).